### ISSUE TITLE
2.0.20 - Change in Selenium's RemoteWebElement conditional statement causes TypeValueIntoTarget to fail.

### ISSUE THREAD

**Issue by timothyjmtan** [serenity-bdd/serenity-core#1475]
Hello,

For serenity version 2.0.20, there is an issue with the `TypeValueIntoTarget` class.

In line 23:
`this.target.resolveFor(theUser).sendKeys(this.getFollowedByKeys());`

In some cases (well, most cases for me actually) the method `this.getFollowedByKeys()` returns an empty string, i.e. ""

In Selenium's `RemoteWebElement` class, in the `sendKeys() method`, there is a modification in their conditional statement that causes this to be an issue.

**In 1.9.36 -** 
```
public void sendKeys(CharSequence... keysToSend) {
        if (keysToSend == null) {
            throw new IllegalArgumentException("Keys to send should be a not null CharSequence");
        } else {
            ...   
        }
    }
```

**In 2.0.20 -** 
 ```
public void sendKeys(CharSequence... keysToSend) {
     if (keysToSend != null && keysToSend.length != 0) {
        ...
      } else {
            throw new IllegalArgumentException("Keys to send should be a not null CharSequence");
      }
```

The additional check in the conditional statement i.e. the `keysToSend.length != 0` will return false and thus the `else` statement will be executed.

To fix, my proposal would be to add in a conditional statement in `TypeValueIntoTarget` class to only execute line 23 if this.getFollowedByKeys() isn't empty.

```
    @Step("{0} enters '#theText' into #target")
    public <T extends Actor> void performAs(T theUser) {
        this.target.resolveFor(theUser).sendKeys(new CharSequence[]{this.theText});
        if(this.getFollowedByKeys().length != 0) {
             this.target.resolveFor(theUser).sendKeys(this.getFollowedByKeys());
        }
    }
```

**Comment by wakaleo** [comment]
Well spotted - could you propose a PR?

**Comment by timothyjmtan** [comment]
Sure, will get back to you when I reach home tonight. My corporate environment doesn't allow me to do PRs into github repos.

**Comment by wakaleo** [comment]
Fixed in 2.0.25
